pkgs/applications/office/spice-up/0001-Remove-environment-dependant-CMake-configuration.patch

```diff
@@ -0,0 +1,46 @@
+From d6bfe4f4584a575209e790cc31756411925c73f6 Mon Sep 17 00:00:00 2001
+From: Sam Parkinson <sam@sam.today>
+Date: Sat, 3 Feb 2018 13:16:16 +1100
+Subject: [PATCH] Remove environment dependant CMake configuration
+
+This is broken as nix unsets the $USER environment variable
```

Comment:
Would setting the variable fix this? We should also set patch upstream, would quoting the variables work?